docs-gh-pages/consensus.md

```diff
@@ -0,0 +1,91 @@
+## Consensus
+
+### Architecture
+Every new `CheckpointBlock` is being created under a consensus round.
+
+### Glossary
+- **consensus owner** - the node which sent consensus proposal to others and initiated a new round
+- **facilitator** - node which has been selected to take part in a new round and responded successfully to participate in it (including owner)
+- **SOE** - signed observation edge
```

Comment:
Maybe SOE could be explained more? Also we could have a separate section with an explanation of every domain term e.g. checkpoint block, tip, snapshot. We could then use these terms in the documentation and have them linked to a more thorough explanation.

Reply:
yeah, actually I was thinking about separate page for glossary or domain entities terms, and then reference here.